demo/demo.md

```diff
@@ -0,0 +1,117 @@
+<!--
+The demo.md file is a compiled document. No edits should be made directly to this file.
+README.md is created by running `npm run build:docs`.
+This file is generated based on a template fetched from `./docs/partials/demo.md`
+-->
+
+# Pane
+
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./description.md) -->
+<!-- The below content is automatically added from ./description.md -->
+`<auro-pane>` is a [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) displaying selectable shoulder dates and associated prices.
+<!-- AURO-GENERATED-CONTENT:END -->
+
+## auro-pane use cases
+
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./useCases.md) -->
+<!-- The below content is automatically added from ./useCases.md -->
+The `<auro-pane>` element should be used in situations where users may:
+
+* Select a date to see available flight options
+* See multiple dates with price and availability information
+<!-- AURO-GENERATED-CONTENT:END -->
+
+## Example(s)
+
+## Basic Example
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/basic.html -->
+  <auro-pane date="2020-09-20"></auro-pane>
+  <auro-pane date="2020-09-21" selected></auro-pane>
+  <auro-pane date="2020-09-22" disabled></auro-pane>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/basic.html -->
+
+```html
+<auro-pane date="2020-09-20"></auro-pane>
+<auro-pane date="2020-09-21" selected></auro-pane>
+<auro-pane date="2020-09-22" disabled></auro-pane>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Date and Price
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/basic.html -->
+  <auro-pane date="2020-09-20"></auro-pane>
+  <auro-pane date="2020-09-21" selected></auro-pane>
+  <auro-pane date="2020-09-22" disabled></auro-pane>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/basic.html -->
+
+```html
+<auro-pane date="2020-09-20"></auro-pane>
+<auro-pane date="2020-09-21" selected></auro-pane>
+<auro-pane date="2020-09-22" disabled></auro-pane>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Small
```

Comment:
```suggestion
### Use sm size

Use the `sm` attribute to main the small UI that is consistent with mobile and desktop.
```

Reply:
Old documentation language is based on content when we didn't have an API page to explicitly call out each API attribute and property. The demo page can be more generic as we have done on other components.

Reply:
Disagree. The main demo page should have descriptions as to what the demo is illustrating back to the user. This is how the demo pages always have been done. The API page is to go into additional detail and more display options. 

Please add in the descriptive text as to the demo

```
Use the `sm` attribute to main the small UI that is consistent with mobile and desktop.
```